napari/components/_layers/view/main.py

```diff
@@ -76,12 +135,12 @@ def dragLeaveEvent(self, event):
             self.vbox_layout.itemAt(i).widget().setSelected(False)
 
     def dragEnterEvent(self, event):
-        event.accept()
         divs = []
         for i in range(0, self.vbox_layout.count(), 2):
             widget = self.vbox_layout.itemAt(i).widget()
             divs.append(widget.y()+widget.frameGeometry().height()/2)
         self.centers = [(divs[i+1]+divs[i])/2 for i in range(len(divs)-1)]
+        event.accept()
```

Comment:
just out of curiosity, does this change have any actual effect on the program?

Reply:
I have removed the unnecessary `event.accept` that was in the mouse move / press / release events, but this `event.accept` which is inside the drag enter event is necessary to allow the drag to register. Without it the dividers between the layers don't change color